src/components/Search/Search.tsx

```diff
@@ -97,9 +98,16 @@ const Search: React.FC = () => {
 			<section id="values-options" className="options">
 				{makeOption(state.values, "values")}
 			</section>
-			<button className="cta-button" onClick={runSearch}>
-				Search
-			</button>
+			<OpenMenuContext.Consumer>
+				{({toggleMenu}) => (
+					<button className="cta-button" onClick={(event) => {
```

Comment:
I think at some point we're gonna need to flesh this button's `onClick` out even more; I went ahead and created #28